ppb/systems/renderer.py

```diff
@@ -202,8 +202,20 @@ def render_background(self, scene):
         )
         sdl_call(SDL_RenderClear, self.renderer, _check_error=lambda rv: rv < 0)
 
+    def is_visible(self, game_object):
+        if hasattr(game_object, 'width') and game_object.width <= 0:
```

Comment:
Would it be valuable to also exclude Sprites without `__image__` or `__image__() == None` here?

Reply:
Ehhhh..... Maybe if we expected it to be reused? I'd honestly would rather just rename it, since it's a single-use helper and the one call site immediately calls `__image__()` right after.

Reply:
But doesn't it also check DoNotRender there? We could bail earlier.

Reply:
Oh, right, the only difference would be if it didn't have `__image__`.